mods/ra/maps/allies-13/rules.yaml

```
@@ -0,0 +1,32 @@
+World:
+	LuaScript:
+		Scripts: allies13.lua, campaign.lua, utils.lua
+	MissionData:
+		BackgroundVideo: spy.vqa
+		WinVideo: apcescpe.vqa
+		LossVideo: bmap.vqa
+		Briefing: LANDCOM 16 HQS.\nTOP SECRET.\nTO: FIELD COMMANDER A9\n\nCONGRATULATIONS. CAPTURING TECH CENTERS HAS REVEALED AN UNDERGROUND WEAPONS FACILITY. PLACE EXPLOSIVE CHARGES ON ALL GENERATORS. RESULTING EXPOSIONS SHOULD DESTROY FACILITY.\nGET OUT BEFORE NERVE GAS IS USED.\n\nTRANSMISSION ENDS.
+	ScriptLobbyDropdown@difficulty:
+		ID: difficulty
+		Label: dropdown-difficulty.label
+		Description: dropdown-difficulty.description
+		Values:
+			easy: options-difficulty.easy
+			normal: options-difficulty.normal
+			hard: options-difficulty.hard
+		Default: normal
+	TimeLimitManager:
+		CountdownLabel: MISSION_TEXT
+		CountdownText: Sarin gas released in {0}
+
```

```suggestion
	StartGameNotification:
		Notification: MissionTimerInitialised

```
If you wished to match the original opening speech, or maybe `TimerStarted` if you'd prefer brevity.